nixos/modules/services/web-apps/netbox.nix

```diff
@@ -103,19 +103,17 @@ in
       type = lib.types.package;
       default =
         if lib.versionAtLeast config.system.stateVersion "25.05" then
```

Comment:
This is inconsitent with the defaultText, and here it's wrong: Here, `25.11` should be used.